connectors/opa/main.go

```diff
@@ -62,6 +77,26 @@ func RunCmd() *cobra.Command {
 			router.Use(gin.Logger())
 
 			bindAddress := fmt.Sprintf("%s:%d", ip, port)
+			var client kclient.Client
+			if utils.GetisTLSPort() {
```

Comment:
it should be useTLS 
We talked that we can create a single server initialization, if Certificates and CA env variables are set, we add them to the server, independently of use or not TLS